python/paddle/distribution/laplace.py

```diff
@@ -52,8 +58,10 @@ class Laplace(distribution.Distribution):
                 1.31554604)
 
     """
+    loc: Numberic
+    scale: Numberic
```

Comment:
```suggestion
    loc: Tensor
    scale: Tensor
```

实例化之后，`loc scale` 的值都转换为了 `Tensor`